scraper.rb

```diff
@@ -1,3 +1,55 @@
-require File.join(File.dirname(__FILE__), "planningalerts_xml_feed", "planningalerts_xml_feed")
+require 'scraperwiki'
+require 'mechanize'
+
+agent = Mechanize.new
+
+def extract_from_p(contents, name)
+	contents.at('p:contains("'+name+'")').inner_text.split(name).last.strip
+end
+
+def scrape_page(page)
+	contents = page.search('div.span6')
+	record = {
+		"info_url" => page.uri.to_s,
+		"council_reference" => contents.search('h1').inner_text,
+		"comment_url" => page.uri.to_s,
+		"applicatant" => extract_from_p(contents,"Applicant:"),
+		"description" => contents.at('p:contains("Location:")').next_element.inner_text,
+		"address" => extract_from_p(contents,"Location:"),
+		"date_scraped" => Date.today.to_s,
+		"date_closing" => extract_from_p(contents,"Advertising closes ")
```

Comment:
This string should be parsed into a date or at least a string in the "YYYY-MM-DD" format.